Dockerfile

```diff
@@ -37,6 +37,10 @@ WORKDIR /kvrocks
 
 COPY --from=build /kvrocks/build/kvrocks ./bin/
 
+ARG TARGETARCH
+COPY tools/redis-cli-${TARGETARCH} /usr/bin/redis-cli
+RUN chmod a+x /usr/bin/redis-cli
```

Comment:
IMHO if a user run `docker build` towards this dockerfile in him/her device instead of CI, a fatal error will appear since there is no `tools` dir.

So I think It is not a reasonable change.

Reply:
And I want to emphasize my point: the dockerfile cannot have different behavior between built by a user and triggered in CI.

Reply:
done